libclc/libspirv/lib/amdgcn-amdhsa/group/collectives.cl

```diff
@@ -382,7 +378,7 @@ long __clc__3d_to_linear_local_id(ulong3 id) {
   _CLC_DEF _CLC_OVERLOAD _CLC_CONVERGENT TYPE __spirv_GroupBroadcast(          \
       int scope, TYPE x, ulong local_id) {                                    \
     if (scope == Subgroup) {                                                   \
-      return _Z28__spirv_SubgroupShuffleINTELI##TYPE_MANGLED##ET_S0_j(         \
+      return __spirv_SubgroupShuffleINTEL(                                     \
```

Comment:
The formatting here looks a bit off. Running clang-format might trigger a lot of surrounding changes, which would be noisy. Maybe at least we could keep this and `x, local_id);` all on one line?